photon-client/src/test/scala/com/linkedin/photon/ml/data/avro/AvroDataReaderTest.scala

```diff
@@ -147,9 +147,9 @@ object AvroDataReaderTest {
     .record("testNameAndTermSchema")
     .namespace("com.linkedin.photon.ml.avro.data")
     .fields()
-    .name(AvroFieldNames.NAME).`type`().stringType().noDefault()
-    .name(AvroFieldNames.TERM).`type`().stringType().noDefault()
-    .name(AvroFieldNames.VALUE).`type`().doubleType().noDefault()
+    .name(ResponsePredictionFieldNames.NAME).`type`().stringType().noDefault()
+    .name(ResponsePredictionFieldNames.TERM).`type`().stringType().noDefault()
+    .name(ResponsePredictionFieldNames.VALUE).`type`().doubleType().noDefault()
```

Comment:
These tests don't actually take either of the two types of input Avro schemas Photon handles as input, so they should use AvroFieldNames instead.